### DISCUSSION TITLE
Badges in Karazhan (again)

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#6480] [Suggestions]
It was recently made public that Karazhan is going to be having its attunement no longer required due to people having issues finding groups to get attuned. This is a good change but I think it only fixes part of the problem, the problem was that people couldn't get dungeon groups together in order to complete their attunement. With that being gone a lot of players can do kara and get really good gear for t4. The issue  is badge items are still really good some of them are better than kara items, and for items like Bloodlust Brooch, Shred Idol, On use Spell Power Trinkets, On use block rating trinket. These items can only be acquired by badges. So because kara doesn't drop badges they more or less cannot get these items because as stated earlier they cannot find dungeon groups in order to get them.
For reference I have a shaman and I have good enough gear to do Karazhan and I'm fortunate enough to have done the attunement while people still ran dungeons. But I still cant get on the on use spell power trinket because I sit 2 hours in queue as a healer
I truly think, for the betterment of the new players that come in. Karazhan badges should be added sooner rather than later. Its only hurting the new players which are the players that breath new life into the server.

**Comment by 55Honey** [comment]
Hello Never6,
Thanks for your suggestion.
The planned progression is to release badges in Kara when Zul Aman is released. You can find a post mentioning that pinned in the Karazhan channel of our discord.

Regards

**Reply by Never6** [reply]
Yes, im well aware. I think that is the problem

**Comment by balleny** [comment]
> So because kara doesn't drop badges they more or less cannot get these items because as stated earlier they cannot find dungeon groups in order to get them.

i am queueing and completing random heroics on chars in t1 / t2 gear via rdf without any issues, especially since the dampening has actually been removed.

if you would allow boj to drop outisde of heroics right now, you would affect the later boj item releases, and more importantly the gem market.
I guess in the first place, your suggestion should cover also the impact and possible mitigations. That at least could increase the chances for the suggestion to find their place.
(similar to how it was proposed and declined three times already to move ZA in between T5 and T6 releases

**Comment by Never6** [comment]
> i am queueing and completing random heroics on chars in t1 / t2 gear via rdf without any issues, especially since the dampening has actually been removed.
>
completing them was never the issue its getting them to pop in the first place, at certains times of the day this is near impossible. due to lack of tanks and healers (even dps in some cases 

> if you would allow boj to drop outisde of heroics right now, you would affect the later boj item releases, and more importantly the gem market. I guess in the first place, your suggestion should cover also the impact and possible mitigations. That at least could increase the chances for the suggestion to find their place. (similar to how it was proposed and declined three times already to move ZA in between T5 and T6 releases

whats the difference between that and players doing heroics right now? there isnt one.

**Reply by balleny** [reply]
reply function ftw.

If you drop badges in raids, people have even less incentive to run heroics to stock up on badges for upcoming item releases.

**Reply by Never6** [reply]
but you said that releasing them now would affect later BoJ releases and the gem market.
How if people are farming badges anyway, there is zero difference. They get more,  but they cap out at 200 afaik. 
Moving ZA release to in-between T5 and T6 would be also be good. Side tangent but I really don't see the purpose of keeping the server blizzlike. This is past content now but in original vanilla Dire Maul wasnt out on release and came later, but on this server it was out on 59 release, nobody cared. You can do things differently than blizzard but still have blizzlike scripts.
I really don't think if u gave badges in raids people would have less incentive. Cause it would only be kara, 10 per week.
If you really want to farm badges doing heroics is still better

**Comment by Nyeriah** [comment]
Accepted, badges will be dropping in all raids starting from next lockout.
